modules/jar-handling-utils/pom.xml

```diff
@@ -0,0 +1,117 @@
+    <version>4.5.0-SNAPSHOT</version>
+</parent>
+
+<artifactId>mule-jar-handling-utils</artifactId>
+<packaging>jar</packaging>
+<name>Mule jar handling utils</name>
+<description>
+    This is a multi release module that allows handling of JDK internal jar classes needed for Java 8 runtime and a cleaner way in 11+.
+</description>
+
+<properties>
+    <formatterConfigPath>../../formatter.xml</formatterConfigPath>
+</properties>
+    <build>
+        <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <version>${mavenCompilerVersion}</version>
+            <executions>
+                <execution>
+                    <id>default-compile</id>
+                    <goals>
+                        <goal>compile</goal>
+                    </goals>
+                    <configuration>
+                        <release combine.self="override"></release>
+                        <source>${javaVersion}</source>
+                        <target>${javaVersion}</target>
+                        <!-- recompile everything for target VM except the module-info.java -->
+                        <excludes>
+                            <exclude>module-info.java</exclude>
+                        </excludes>
+                    </configuration>
+                </execution>
+                <execution>
+                    <id>base-compile</id>
+                    <configuration>
+                        <release>11</release>
+                        <compileSourceRoots>
+                            <compileSourceRoot>${project.basedir}/src/main/java11</compileSourceRoot>
+                        </compileSourceRoots>
+                        <multiReleaseOutput>true</multiReleaseOutput>
+                    </configuration>
+                </execution>
+                <execution>
+                    <id>module-info-compile</id>
+                    <goals>
+                        <goal>compile</goal>
+                    </goals>
+                    <configuration>
+                        <release>11</release>
+                        <!-- recompile the module-info.java -->
+                        <includes>
+                            <include>module-info.java</include>
+                        </includes>
+                    </configuration>
+                </execution>
+            </executions>
+            <configuration>
+                <encoding>ISO-8859-1</encoding>
+                <proc>none</proc>
+            </configuration>
+        </plugin>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-jar-plugin</artifactId>
+            <configuration>
+                <archive>
+                    <manifestEntries>
+                        <Multi-Release>true</Multi-Release>
+                        <Automatic-Module-Name>mule.jar.handling.utils</Automatic-Module-Name>
```

Comment:
this is already configured in the parent, just need to set the property `javaModuleName`

Reply:
sure